tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -434,8 +434,11 @@ generate_qemu_options() {
 		qemu_options+=(functionality:--enable-libpmem)
```

Comment:
Should we remove `qemu_options+=(functionality:--enable-libpmem)` as we are calling `qemu_options+=(functionality:--disable-libpmem)` right after, which is a bit confusing?

Reply:
Ah, this is done in a later commit. I guess that implicitly seconds Greg suggestion to move it :)